client/auth/action-creators.ts

```diff
@@ -128,7 +127,7 @@ export function logOut() {
     const result = await fetchJson<void>(apiUrl`sessions`, {
       method: 'delete',
     })
-    sessionStorage.clear()
+
```

Comment:
The point of doing this before was to clear the message storage specifically, so you can't just delete this now. You need to preserve the existing semantics of the stuff that was previously using sessionStorage, namely, it all needs to get cleared when the current user changes.

(This is done in 2 places, here and the other line you deleted below, so make sure to deal with it in both)

Reply:
Another alternative is to use the current user ID as part of the map key. In any case, you need to preserve the existing behavior here